apps/chat/src/backend/serverless/template.yaml

```diff
@@ -1010,7 +1016,7 @@ Resources:
                 statusCode: 200,
                 headers: {
                   'Access-Control-Allow-Headers': 'Authorization',
-                  'Access-Control-Allow-Origin': 'https://localhost:9000',
+                  'Access-Control-Allow-Origin': '*',

```

Comment:
Recommend adding an instruction in README because all these Lambda functions will be open to public.

e.g., In a production environment, we recommend that you use a specific origin for the Access-Control-Allow-Origin response header.

Reply:
Good idea. Added comment on the template.yaml instead, to not clutter the readme with specifics

Reply:
This also breaks credential exchange authorization :-(